src/sentry/coreapi.py

```diff
@@ -172,18 +174,30 @@ def insert_data_to_database(self, data, start_time=None,
 
         cache_timeout = 3600
         cache_key = cache_key_for_event(data)
-        default_cache.set(cache_key, data, cache_timeout)
 
         # Attachments will be empty or None if the "event-attachments" feature
         # is turned off. For native crash reports it will still contain the
         # crash dump (e.g. minidump) so we can load it during processing.
         if attachments is not None:
             attachment_cache.set(cache_key, attachments, cache_timeout)
 
-        task = from_reprocessing and \
-            preprocess_event_from_reprocessing or preprocess_event
-        task.delay(cache_key=cache_key, start_time=start_time,
-                   event_id=data['event_id'])
+        if features.has('projects:kafka-preprocess', project=project):
```

Comment:
Are there any potential deploy-time issues here where the feature hasn't been registered properly yet or something, and we end up slamming PG constantly looking for it  and/or using the wrong default? I seem to remember this being an issue with another deploy.

Reply:
Yes. Because we don’t have negative caching yet.

Reply:
Would it not fall back to the default (`False`) though? Is there something I need to do code-wise to mitigate this?